oauth2/oauth2-client/src/main/resources/org/springframework/security/oauth2/client/oauth2-client-schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE oauth2_authorized_client (
```

Comment:
I would not be shy about making values larger rather than shorter. Storage space is typically not an issue these days. I'd consider making the token value and the refresh token value's both blobs since we should not ever need to search by them.

Reply:
I made both token columns `blob` and increased the size for a couple of other columns too. This is now merged.